operator/src/main/resources/Operator.properties

```diff
@@ -187,7 +187,7 @@ WLSKO-0175=Job {0} in namespace {1} failed with status {2}. Check log messages \
   copied from the introspector pod {3} log for additional information.
 WLSKO-0176=Job {1} in namespace {0} failed, job details are {2}
 WLSKO-0177=Pod {0} in namespace {1} failed, the pod status is {2}
-WLSKO-0180=Operator cannot start. 'Dedicated' namespace strategy selected, but the CRD is not installed
+WLSKO-0178=Operator cannot start. Operator 'domainNamespaceSelectionStrategy' is set to 'Dedicated', but the Custom Resource Definition for ''domains.weblogic.oracle'' is not installed.
```

Comment:
Is there a particular reason to renumber this? I deliberately skipped some numbers to allow 178 and 179 for messages that seemed logically connected to the previous ones. Do we have a rule banning skipped numbers?

Reply:
Sorry, I had two log messages 0178 and 0179 in my previous version of the changes. Now they are gone and I thought yours could move up. I assume that we could skip numbers.